core/arch/arm/plat-k3/main.c

```diff
@@ -49,6 +49,15 @@ void console_init(void)
 	register_serial_console(&console_data.chip);
 }
 
+static TEE_Result console_runtime(void)
+{
+	/* Console is only used for errors after this point */
+	trace_set_level(TRACE_ERROR);
```

Comment:
Isn't this a bit inconvenient when doing development? Wouldn't a build-time conditional (`if IS_ENABLED(CFG_FOO)`) help?

Reply:
Sure, I'll add a build flag to disable this feature when debugging.